_posts/2016-09-14-supercharged-seo-with-drupal-8.md

```diff
@@ -0,0 +1,118 @@
+
+## What is SEO?
+
+Search Engine Optimization (SEO) is a marketing discipline focused on optimizing a website’s architecture and content so that it performs well (read “ranks high”) in organic search engine results.
+
+Search engines (Google, Bing, etc.) are changing their search algorithms many times throughout the year. There are over 200 ranking factors that become updated with every algorithmic change. Having said that, we begin to realize that there are many techniques that website owners can use to optimize their digital property for search engine consumption. So where do we start? How do we know what efforts will bring us the best ROI and let our marketers do their job effectively in the long run?
+
+## SEO Outlook 2017
+
+The Savas Labs team stays dialed in on the current trends of Search Engines Optimization. Our own experiences with SEO, trials and errors and aggregated research data helped us develop sound judgment and solid knowledge of the current marketing landscape.
+
+Here are 4 ranking factors that we identified as being most important as of Q4 2016. Our forecast is that those 4 factors will likely remain at the top of the list throughout 2017.
+
+#### 1. Content
+
+Content is still king! Yes, that’s right. It is and it will always be! You’ve got to be relevant in order to even appear in search. And nothing will make you more relevant than carefully crafted, practical, awesome, juicy, shareable, actionable (you name it) CONTENT!
+
+#### 2. Backlinks
+
+Not just any good ‘ol link to your website. [Good backlinks](http://cognitiveseo.com/blog/3057/the-definition-of-the-high-quality-backlink/) come from high authority domains that are in the same niche as your website. Strong backlinks bring quality traffic and are therefore considered highly desirable to your SEO cause.
+
+#### 3. Responsive Design
+
+With more people using their handheld devices to browse the internet, it has become increasingly important to make a website look good across multiple platforms (smartphone, tablet, etc.). It is not an option in 2017 - it is a necessity! While we won’t get into the notoriously labeled [Google algorithm update “Mobilegeddon”](https://webmasters.googleblog.com/2015/04/rolling-out-mobile-friendly-update.html) that happened in April 2015, we will provide some interesting statistics to back up the importance of responsive design.
+
+> “There are more mobile internet users than desktop internet users; 52.7% of global internet users access the internet via mobile, and 75.1% of U.S. internet users access the internet via mobile.”
+
+> “4 out of 5 consumers use a Smartphone to shop.”
+
+#### 4. Page Speed
+
+In response to the substantial increase in mobile traffic growth, search engines have acknowledged the importance of page speed and the effect it has on user experience (UX) and now give more weight to fast-loading websites.
+
+> “40% of people abandon a website that takes more than 3 seconds to load.”
```

Comment:
Source?
Drop the quotation marks from the text since they're in the styled output